apps/minifront/src/components/ibc/layout.tsx

```diff
@@ -1,18 +1,18 @@
 import { Card } from '@penumbra-zone/ui/components/ui/card';
-import { IbcOutForm } from './ibc-out-form';
-import { IbcInForm } from './ibc-in-form';
+import { IbcOutForm } from './ibc-out/ibc-out-form';
+import { IbcInForm } from './ibc-in/ibc-in-form';
 import { ArrowLeftIcon, ArrowRightIcon } from '@radix-ui/react-icons';
 
 export const IbcLayout = () => {
   return (
     <>
       <div className="fixed inset-0 z-[-100] size-full bg-[url('penumbra-logo.svg')] bg-[length:160vmax] bg-fixed bg-[top_50%_left_25vw] bg-no-repeat" />
       <div className='flex flex-1 flex-col gap-4 md:flex-row md:place-content-around'>
-        <Card light className='relative overflow-visible md:self-start'>
+        <Card light className='relative overflow-visible md:self-end'>
           <ArrowRightIcon className='absolute inset-y-0 right-0 z-10 my-auto -mr-14 size-16 text-stone-300' />
           <IbcInForm />
         </Card>
-        <Card gradient className='relative overflow-visible md:self-end'>
+        <Card gradient className='relative overflow-visible md:self-start'>
```

Comment:
Given the ibc chain side will grow to be much longer than the ibc out, put the right side to be on top so it doesn't get pushed down.

<img width="1725" alt="Screenshot 2024-04-25 at 3 52 30 PM" src="https://github.com/penumbra-zone/web/assets/16624263/1ddb73ff-5018-47f5-946b-329db32d955d">

Reply:
the downward placement was a significant part of the design concept, providing an intuitive sense of descending into a place of concealment. cc @hdevalence 

if you used my style code, on wide enough viewports it should not compete for vertical space with the "Shield Assets" card - it will be pushed to the bottom of the screen, no more. 

on narrower viewports that cannot accomodate the cards side-by-side, it responsively switches to a column layout. the order for that is configurable much like the other pages

Reply:
the design thought of the original concept was that when a user clicks on "Shield Funds" it's probably because they don't already have funds on Penumbra and so the first thing that should appear is the deposit dialog.

the reason to offset the two cards was so that the arrows could extend "into" the other domain, without pointing directly at the other card, rather than a sense of descent. the idea was just that the deposit card would have an arrow that goes into the inside of the eclipse. the thing that actually matters was having a two-column layout where one column is inside the eclipse and the other is outside.

> Given the ibc chain side will grow to be much longer than the ibc out

i think i'm missing something, i'm not sure why the length matters here? we can just have two columns, no?

Reply:
cc @bibnub

Reply:
prev context https://github.com/penumbra-zone/web/issues/800

Reply:
Helpful context. With the flexbox start/end layout, as the "start" element grew beyond the size of the window (not shown in the screenshot), the "end" element got pushed below view. Swapping them was a fix, but I see it doesn't fix the concept so well. The easiest solution is simply to add top margin to the right side.

<img width="1718" alt="Screenshot 2024-04-26 at 2 53 34 PM" src="https://github.com/penumbra-zone/web/assets/16624263/553527be-3f5e-4bbd-80c5-47cb774ed97c">

We may need to evaluate these arrows again if the left side box more space than given.